### ISSUE TITLE
Strings containing braces might cause an indefinite hang

### ISSUE THREAD

**Issue by jszczepanski** [f5devcentral/f5-automation-config-converter#79]
### Environment
 * ACC Version: 1.18
 * AS3 Version: N/A
 * BIG-IP Version: N/A

### Summary
I've encountered this fun quoting corner case a while ago in a client UCS, only now tried to run it through ACC.

Brace matching might be broken in some strings-containing-braces scenarios, resulting in an indefinite freeze. Might be related to #78 

### Steps To Reproduce
1. Convert any of the following configs:
```conf
ltm data-group internal /Common/regexp_special_characters {
    records {
        " " {
            data LostInSpace
        }
        "\\{" {
            data BracketStart
        }
        "\\}" {
            data BracketEnd
        }
    }
    type string
}
```
```conf
ltm data-group internal /Common/regexp_restore_characters {
    records {
        BracketEnd {
            data }
        }
        BracketStart {
            data {
        }
        LostInSpace {
            data " "
        }
    }
    type string
}
```

2. ACC is now frozen indefinitely.

Note that unquoted strings `{` and `}` seemingly might also appear in other places in config; unsure if in all string-type fields.
```
[root@localhost:LICENSE EXPIRED:Standalone] config # tmsh modify auth user admin description "\{"    
[root@localhost:LICENSE EXPIRED:Standalone] config # tmsh list auth user admin
auth user admin {
    description {
    encrypted-password 123
    partition Common
    partition-access {
        all-partitions {
            role admin
        }
    }
    session-limit -1
    shell none
}
```

**Comment by delgadillo22** [comment]
Thank you for the report, I've added this to our internal backlog as CHARON-587 and will get this prioritized.

**Comment by p-semenov-f5** [comment]
Fixed in 1.19
